content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst

```diff
@@ -110,23 +111,33 @@ specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a
 specified, a *Pass - Fail* quality check is created for that specific work order, rather than the
 manufacturing order as a whole.
 
-*Pass - Fail* quality checks configured for work orders must be processed from the tablet view. To
-do so, begin by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing
-Orders`. Select a manufacturing order that includes a work order for which a quality check is
-required. Open the tablet view for that work order by selecting the :guilabel:`Work Orders` tab, and
-then clicking the :guilabel:`📱 (tablet)` button on the order's line.
+*Pass - Fail* quality checks configured for work orders must be completed from the *Shop Floor*
```

Comment:
If you happen to review this PR before seeing my comments in the [instructions quality check PR](https://github.com/odoo/documentation/pull/6779#discussion_r1436558331), here's a recap:

- we use italics when defining concepts or Odoo terms for the very first time. 
- However, the times after, we probably don't need to capitalize pass-fail anymore. However, each app and term is different, so if you strongly disagree with that, feel free to let me know :smile: